### PR TITLE
Added method to check for invalid properties in a hook

### PR DESCRIPTION
**Pull request by jarvis-666**

### CODE REVIEW

File: tcadmin/resources/hook.py [hunk 1]
```diff
@@ -101,3 +101,12 @@ def to_api(self):
             "task": self.task,
             "triggerSchema": self.triggerSchema,
         }
+
+    def check_hook_validity(self):
```
Comment (djmitche) [inline tcadmin/resources/hook.py:105]: When does this function get called?

Comment (jarvis-666) [reply tcadmin/resources/hook.py:105]: Hey, have been trying to figure the entry-point for it. I found create_hook() and update_hook() in tcadmin/update.py. Can you please guide me?

**Comment by petemoore** [comment]
@jarvis-666 Are you still intending to work on this? Many thanks.

**Comment by leplatrem** [comment]
@jarvis-666 Should we close this PR?

**Pull request closed without merging**